lib/codegen/meta/cdsl/predicates.py

```diff
@@ -262,6 +262,21 @@ def __init__(self, field, value):
         self.value = value
 
 
+class IsZero(FieldPredicate):
+    """
+    Instruction predicate that checks if an immediate instruction format field
+    is equal to zero.
+
+    :param field: `FormatField` to be checked.
+    :param value: The constant value to check.
+    """
+
+    def __init__(self, field):
+        # type: (FormatFiend) -> None
+        # FIXUP: Is this calling `super` correctly?
```

Comment:
Wasn't totally sure about the type comment here, and wanted to make sure this looked alright to you.